Patch-Source: https://github.com/ueno/libkkc/commit/ce17a35d3dca32706ae2dd48c7859a36531a9b59.patch
--
From ce17a35d3dca32706ae2dd48c7859a36531a9b59 Mon Sep 17 00:00:00 2001
From: Mamoru TASAKA <mtasaka@fedoraproject.org>
Date: Thu, 22 Aug 2024 00:53:12 +0900
Subject: [PATCH] Fix invalid escape on json file
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix the following invalid escape on
data/rules/default/rom-kana/default.json :

/usr/share/libkkc/rules/default/rom-kana/default.json:121:47: 解析エラー: unexpected
character `,', expected string constant
---
 data/rules/default/rom-kana/default.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/rules/default/rom-kana/default.json b/data/rules/default/rom-kana/default.json
index a4616fa..fd10680 100644
--- a/data/rules/default/rom-kana/default.json
+++ b/data/rules/default/rom-kana/default.json
@@ -120,7 +120,7 @@
             "myo": ["", "みょ" ],
             "myu": ["", "みゅ" ],
             "n": ["", "n", "n", "ん", "ン" ],
-            "n\'": ["", "ん" ],
+            "n'": ["", "ん" ],
             "na": ["", "な" ],
             "ne": ["", "ね" ],
             "ni": ["", "に" ],
